### PR TITLE
7.0 travel purchase alert

### PR DESCRIPTION
**Pull request by joaoalf**
Send alert when a PO linked to the travel is opened

Depends on #20

### CODE REVIEW

File: travel_purchase/purchase_order.py [hunk 1]
```diff
@@ -21,10 +21,28 @@
...
 class purchase_order(orm.Model):
     _inherit = 'purchase.order'
+
+    def _get_responsible_emails(
+            self, cr, uid, ids, name=None, args=None, context=None):
+        context = context or {}
+        res = {}
+
+        for purchase in self.browse(cr, uid, ids, context=context):
+            if purchase.state == 'approved' and purchase.travel_id is not False:
```
Comment (bwrsandman) [inline travel_purchase/purchase_order.py]: This line is too long

File: travel_purchase/purchase_order.py [hunk 2]
```diff
@@ -21,10 +21,28 @@
 ##############################################################################
 
 from openerp.osv import orm, fields
+from .res_config import get_alert_address
```
Comment (bwrsandman) [inline travel_purchase/purchase_order.py]: Where is this function, it's failing on travis?

try:

``` python
from openerp.addons.travel.res_config import get_alert_address
```

**Comment by joaoalf** [comment]
@bwrsandman the problem is that it depends on #20 and it was not merged yet.

**Comment by bwrsandman** [comment]
:+1:

File: travel/i18n/fr.po [hunk 3]
```diff
@@ -36,56 +47,142 @@ msgid "Closed"
...
+#. module: travel
+#: code:addons/travel/travel.py:177
 #, python-format
 msgid "Only draft travels can be unlinked"
 msgstr "Seuls les déplacements en mode brouillon peuvent être supprimés"
 
 #. module: travel
-#: code:addons/travel/travel.py:91
+#: help:travel.passenger,message_unread:0 help:travel.travel,message_unread:0
+msgid "If checked new messages require your attention."
```
Comment (ventilooo) [inline travel/i18n/fr.po:110]: correction : "If checked new messages will require your attention"

**Comment by pedrobaeza** [comment]
As Travis is red and no fix, I close. Feel free to reopen in case there's interest.

**Pull request closed without merging**